asigra.json

```diff
@@ -24,7 +24,8 @@
 	"sysvmsg": "new",
 	"sysvsem": "new",
 	"sysvshm": "new",
-	"vnet": 1
+	"vnet": 1,
+	"nat_forwards": "tcp(80:8787)"
```

Comment:
I don't think this is going to work based on how ds-system discovery is done. Can you please confirm @miwi-fbsd ? I have DHCP issues at the moment and can't test this

Reply:
Good Catch, tested and as expected not working due to the NAT ip.